packages/osd-opensearch/src/cluster.js

```diff
@@ -34,14 +34,27 @@ const execa = require('execa');
 const chalk = require('chalk');
 const path = require('path');
 const { downloadSnapshot, installSnapshot, installSource, installArchive } = require('./install');
-const { OPENSEARCH_BIN, OPENSEARCH_PLUGIN } = require('./paths');
+const { OPENSEARCH_BIN, OPENSEARCH_PLUGIN, OPENSEARCH_SECURITY_INSTALL } = require('./paths');
 const { log: defaultLog, parseOpenSearchLog, extractConfigFiles, decompress } = require('./utils');
 const { createCliError } = require('./errors');
 const { promisify } = require('util');
 const treeKillAsync = promisify(require('tree-kill'));
 const { parseSettings, SettingsFilter } = require('./settings');
 const { CA_CERT_PATH, OPENSEARCH_P12_PATH, OPENSEARCH_P12_PASSWORD } = require('@osd/dev-utils');
 const readFile = util.promisify(fs.readFile);
+const chmodAsync = util.promisify(fs.chmod);
+
+const LATEST_ENGINE_PLUGIN_BASE_URL =
+  'https://ci.opensearch.org/ci/dbc/distribution-build-opensearch';
+
+function generateEnginePluginUrl(version, plugin) {
+  const legacyVersion = `${version}.0`;
+  const [platform, type] =
+    process.platform === 'win32' ? ['windows', 'zip'] : [process.platform, 'tar'];
+  const arch = process.arch === 'arm64' ? 'arm64' : 'x64';
```

Comment:
Do we know that they have darwin releases? Even darwin-arm64?

Reply:
just checked unfortunately i do not get any thing back. im pretty sure the snapshot is available but might be a little bit too much to support non-zips of these plugins.

Reply:
i just removed.